src/igraph/io/objects.py

```diff
@@ -508,35 +508,35 @@ def _export_graph_to_dict_list(
 
     This function is the reverse of Graph.DictList.
 
-    @param use_vids (bool): whether to label vertices in the output data
+    Example:
+
+      >>> g = Graph([(0, 1), (1, 2)])
+      >>> g.vs["name"] = ["apple", "pear", "peach"]
+      >>> g.es["name"] = ["first_edge", "second"]
+
+      >>> g.to_dict_list()
+      ([{"name": "apple"}, {"name": "pear"}, {"name": "peach"}],
+       [{"source": 0, "target": 1, "name": "first_edge"},
+        {"source" 0, "target": 2, name": "second"}])
+
+      >>> g.to_dict_list(use_vids=False)
+      ([{"name": "apple"}, {"name": "pear"}, {"name": "peach"}],
+       [{"source": "apple", "target": "pear", "name": "first_edge"},
+        {"source" "apple", "target": "peach", name": "second"}])
+
+    @param use_vids: (bool) whether to label vertices in the output data
       structure by their ids or their vertex_name_attr attribute. If
       use_vids=False but vertices lack a vertex_name_attr attribute, an
       AttributeError is raised.
-    @param skip_none (bool): whether to skip, for each edge, attributes that
+    @param skip_none: (bool) whether to skip, for each edge, attributes that
```

Comment:
Type annotations should be moved to the function header or removed.

Reply:
yes